src/test/java/com/google/step/datamanager/RestaurantManagerDatastoreTest.java

```diff
@@ -0,0 +1,89 @@
+package com.google.step.datamanager;
+
+import static com.google.step.TestConstants.BLOBKEY_A;
+import static com.google.step.TestConstants.BLOBKEY_B;
+import static com.google.step.TestConstants.RESTAURANT_NAME_A;
+import static com.google.step.TestConstants.RESTAURANT_NAME_B;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.step.model.Restaurant;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class RestaurantManagerDatastoreTest {
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig().setApplyAllHighRepJobPolicy());
+
+  private final RestaurantManagerDatastore restaurantManagerDatastore =
+      new RestaurantManagerDatastore();
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void testCreateRestaurant_success() throws Exception {
+    Restaurant restaurant =
+        restaurantManagerDatastore.createRestaurant(RESTAURANT_NAME_A, BLOBKEY_A);
+    assertEquals(RESTAURANT_NAME_A, restaurant.name);
```

Comment:
It's generally good practice to leave a `\n` before all of your assertions in tests. Please do the same in all the tests in this file.

```suggestion
    
    assertEquals(RESTAURANT_NAME_A, restaurant.name);
```